app/src/main/java/com/github/polypoly/app/menu/ProfileModifyingActivity.kt

```diff
@@ -126,6 +126,7 @@ class ProfileModifyingActivity : ComponentActivity() {
 
     /**
      * The field where the player can write his/her description
```

Comment:
Documentation like this is very good, less pointless // prepended comments would be better !